wemake_python_styleguide/violations/best_practices.py

```diff
@@ -2978,3 +2978,43 @@ class CommentInFormattedStringViolation(TokenizeViolation):
 
     error_template = 'Found comment inside formatted string'
     code = 480
+
+
+@final
+class LeakingForLoopViolation(ASTViolation):
+    """
+    Found a leaking 'for' loop in a class or module body.
+
+    Reasoning:
+        'for' loops in class or module bodies leak iteration variables
+        into the surrounding namespace. This is almost never intended
+        and can lead to confusing bugs.
+
+    Solution:
+        Use list comprehensions, 'map()', or move the logic into a method
+        or helper function. If you must use 'for', explicitly 'del'
```

Comment:
```suggestion
        or helper function. If you must use ``for``, explicitly ``del``
```

we use double backtics for code highlight